readme.md

```diff
@@ -2,23 +2,34 @@
 
 [![Join the chat on Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/7tMENrnv8p)
 
-This project is a work-in-progress effort to improve the integration
-between language server/editors and build tools. This effort is led by the
-Scala Center and Jetbrains, together with the help of other tool developers
-in the Scala ecosystem.
+This project is an effort to improve the integration between language
+server/editors and build tools. This effort is led by the [Scala
+Center](https://scala.epfl.ch/) and [Jetbrains](https://www.jetbrains.com/),
+together with the help of other tooling developers in the Scala ecosystem and
+beyond.
 
 Even though the protocol focuses on Scala developer tools, it's designed to
-be language-agnostic. We're looking for third parties in other programming
-languages to join this effort.
+be language-agnostic.
 
-Read the specification [here](docs/specification.md).
+You can read the specification [here](https://build-server-protocol.github.io/docs/specification).
+
+## Contribution
+
+If you'd like to contribute to the protocol, please check out our
+[CONTRIBUTING](./CONTRIBUTING.md) doc.
 
 ## Maintainers
 
-The current maintainers (people who can merge pull requests) are:
+_Current Maintainers_
 
-* Jorge Vicente Cantero - [@jvican](https://github.com/jvican)
+* Adrien Piquerez - [@adpi2](https://github.com/adpi2)
+* Andrzej Głuszak - [@agluszak](https://github.com/agluszak)
```

Comment:
I added the two of you here since you've both contributed somewhat frequently to the repo lately. Is this alright? Also @abrams27, I know you don't have a ton of commits to the repo, but also know you're heavily involved in the BSP/Jetbrains side of things. Do you also want to be listed here, or would you rather us leave you off?

Reply:
sorry for the delay (last week was really busy), u can add me later if it works for u

Reply:
No worries at all! Feel free to PR yourself in.